package.json

```diff
@@ -30,7 +30,11 @@
     "build": "tsc",
     "clean": "rimraf lib",
     "prepublishOnly": "npm run build",
-    "watch": "tsc -w --listEmittedFiles"
+    "watch": "tsc -w --listEmittedFiles",
+    "format": "pretty-quick --pattern '**/*.{ts,js,tsx,jsx,scss,css,html,md}'",
+    "format:check": "pretty-quick --pattern '**/*.{ts,js,tsx,jsx,scss,css,html,md}' --check",
+    "format:staged": "pretty-quick --pattern '**/*.{ts,js,tsx,jsx,scss,css,html,md}' --staged",
+    "format:check:staged": "pretty-quick --pattern '**/*.{ts,js,tsx,jsx,scss,css,html,md}' --check --staged"
```

Comment:
Maybe change to something like this:
```
"format:check": "npm run format -- --check",
"format:staged": "npm run format -- --staged",
"format:check:staged": "npm run format -- --check --staged"
```

Reply:
I did this in the beginning but I couldn't decide.

We use `jlpm` (aka `yarn`) so I didn't like to use a different package manager (`npm`).
If a developer doesn't have `npm` installed, then they wouldn't be able to execute any of these. Is that correct?

On the other hand, I wouldn't like to force a developer to not use `npm`, and that's the reason why I didn't use `jlpm run format...`.

I'm up for changing it if you think it's best practice.

Reply:
>If a developer doesn't have npm installed, then they wouldn't be able to execute any of these. Is that correct?
I believe `npm` comes always with node.js so we are ok.  But you are correct, how about changing it to  `jlpm format`?

Reply:
After live discussion, we decided to go with extending the scripts using the package manager `jlpm`, since the development of this project requires `jupyterlab` installed and, thus, `jlpm` installed.